src/components/pages/home/index.tsx

```diff
@@ -20,18 +20,47 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
   const [progress, setProgress] = React.useState<any>([])
 
   const topics: any = get(homePageData, 'topics')
+
   const featureDigitalGardening: any = get(
     homePageData,
     'featureDigitalGardening',
   )
   const featureWhatsNew: any = get(homePageData, 'featureWhatsNew')
-  const reactFeatures: any = get(homePageData, 'reactFeatures')
-  const javascriptFeatures: any = get(homePageData, 'javascriptFeatures')
-  const cssFeatures: any = get(homePageData, 'cssFeatures')
-  const reduxFeatures: any = get(homePageData, 'reduxFeatures')
-  const typescriptFeatures: any = get(homePageData, 'typescriptFeatures')
-  const kcdFeatures: any = get(homePageData, 'kcdFeatures')
-  const awsFeatures: any = get(homePageData, 'awsFeatures')
+
+  const reactFeatures = find(homePageData?.homePageFeatures?.features, [
```

Comment:
looking at all these I'm thinking about a function that would look like:

```typescript
const findFeature = (featureName: string) => {
  return find(homePageData?.homePageFeatures?.features, [
    'slug.current',
    featureName
  ]
}
```
This would greatly reduce the repetition so:

```typescript
const reactFeatures = findFeature('react-features')
```